packages/admin/admin-generator/src/commands/generate/generate-command.ts

```diff
@@ -154,21 +154,22 @@ export type VirtualGridColumnConfig<T extends GridValidRowModel> = {
     name: string;
     queryFields?: UsableFields<T, true>[];
     renderCell: (params: GridRenderCellParams<T, any, any>) => JSX.Element;
-} & Pick<GridColDef, "sortBy"> &
+} & Pick<GridColDef<T>, "sortBy"> &
     BaseColumnConfig;
 
 type InitialFilterConfig = {
     items: GridFilterItem[];
     linkOperator?: "and" | "or";
 };
 
+export type GridConfigGridColumnDef<T extends GridValidRowModel> = GridColumnConfig<T> | ActionsGridColumnConfig | VirtualGridColumnConfig<T>;
 export type GridConfig<T extends { __typename?: string }> = {
     type: "grid";
     gqlType: T["__typename"];
     fragmentName?: string;
     query?: string;
     queryParamsPrefix?: string;
-    columns: Array<GridColumnConfig<T> | ActionsGridColumnConfig | VirtualGridColumnConfig<T>>;
+    columns: Array<GridConfigGridColumnDef<T>>;
```

Comment:
The passing of the generic here isn't correct: for GridConfig it's `T extends { __typename?: string }`, for GridConfigGridColumnDef it's `T extends GridValidRowModel`. This only works because GridValidRowModel is basically `Record<string, any>`. We should use the same type restrictions everywhere.